app/views/layouts/_navigation.html.haml

```diff
@@ -7,7 +7,10 @@
         %span.icon-bar
         %span.icon-bar
         %span.icon-bar
-      = link_to (ENV['OSEM_NAME'] || 'OSEM'), root_path, class: 'navbar-brand', title: 'Open Source Event Manager'
+      - if conference.nil? || conference.new_record?
+        = link_to (ENV['OSEM_NAME'] || 'OSEM'), root_path, class: 'navbar-brand', title: 'Open Source Event Manager'
+      - else
+        = link_to "#{conference.organization.name} Organization", organizations_path, class: 'navbar-brand', title: 'Open Source Event Manager'
```

Comment:
I think this should just say `conference.organization.name`, no need to append "Organization". Other than that, LGTM.